src/pagination/pagination.spec.ts

```diff
@@ -519,6 +522,22 @@ describe('ngb-pagination', () => {
       expectPages(fixture.nativeElement, ['« Previous', '1', '+2', '-» Next']);
     });
 
+    it('should render and respond to direction links changes', () => {
+      const html = '<ngb-pagination [collectionSize]="collectionSize" [page]="1" [firstText]="firstText"' +
+          ' [previousText]="previousText" [nextText]="nextText" [lastText]="lastText"' +
+          ' [boundaryLinks]="boundaryLinks"></ngb-pagination>';
```

Comment:
I included in the original commit only the changes for the template string and the more detailed description for the public properties. I can make the rest of the requested changes once you would have a chance to provide additional feedback.